src/serialization.h

```diff
@@ -67,9 +67,9 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 // This represents an uninitialized or invalid format
 #define SER_FMT_VER_INVALID 255
 // Highest supported serialization version
-#define SER_FMT_VER_HIGHEST_READ 28
+#define SER_FMT_VER_HIGHEST_READ 29
```

Comment:
Needs documentation above.

Reply:
"Above"? Is there doc file I need to update?

Reply:
No, I meant [this huge comment](https://github.com/minetest/minetest/blob/63fc728a84a5ba97240233ad1c5d94f1ade2deb1/src/serialization.h#L27-L66)  which needs a new line to document version 29.